R/grabXML.R

```diff
@@ -11,12 +11,13 @@
 #' @export
 #'
 #' @examples
+#'\dontrun{
```

Comment:
just surrounded all the examples that used csw to 'dontrun' as our csw server is no longer actually a thing